kumascript/macros/CSSRef.ejs

```diff
@@ -549,19 +614,22 @@ const text = mdn.localStringMap({
         'Backgrounds_and_borders': '背景和边框',
         'Handling_different_text_directions': '处理不同方向的文本',
         'Overflowing_content': '溢出',
-        'Values_and_units': '值与单位',
+        'CSS_values_and_units': 'CSS values and units',
         'Sizing_items_in_CSS': '在 CSS 中调整大小',
         'Images_media_and_form_elements': '图像、媒体和表单元素',
         'Styling_tables': '样式化表格',
         'Debugging_CSS': '调试 CSS',
         'Organizing_your_CSS': '组织你的 CSS',
+        'Assessment_Fundamental_CSS_comprehension': 'Assessment: Fundamental CSS comprehension',
+        'Assessment_Creating_fancy_letterheaded_paper': 'Assessment: Creating fancy letterheaded paper',
+        'Assessment_A_cool_looking_box': 'Assessment: A cool-looking box',
```

Comment:
@mdn/yari-content-zh Could you please review these `zh-CN` changes?

Reply:
Let's translate it:

```suggestion
        'CSS_values_and_units': 'CSS 值和单位',
        'Sizing_items_in_CSS': '在 CSS 中调整大小',
        'Images_media_and_form_elements': '图像、媒体和表单元素',
        'Styling_tables': '样式化表格',
        'Debugging_CSS': '调试 CSS',
        'Organizing_your_CSS': '组织你的 CSS',
        'Assessment_Fundamental_CSS_comprehension': '测验：基本的 CSS 理解',
        'Assessment_Creating_fancy_letterheaded_paper': '测验：创建精美的信纸',
        'Assessment_A_cool_looking_box': '测验：一个漂亮的盒子',
```